components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam4_resus_mom_soag_tag.in

```diff
@@ -0,0 +1,282 @@
+*     k[uci8] -> k[N2O5] / k[uci8]
+[uci9]   PAN + M -> CH3CO3 + NO2 + M              ; 9.00E-29, 14000
+*     k[uci9] -> k[PAN] / k[uci9]
+[lch3o2_ho2]   CH3O2 + HO2 -> CH3OOH + O2                     ; 4.100E-13, 750
+[lch3o2_no]    CH3O2 + NO -> CH2O + HO2 + NO2                 ; 2.800E-12, 300
+[lch3o2]       CH3O2 + CH3O2 -> 2*CH2O +  2*HO2               ; 9.500E-14, 390
+[lch3ooh]      CH3OOH + OH -> 0.7*CH3O2 + 0.3*CH2O + 0.3*OH   ; 3.800E-12, 200
+[lc2h5o2_no]   C2H5O2 + NO -> CH3CHO + HO2 + NO2              ; 2.600E-12, 365
+[lc2h5o2]      C2H5O2 + C2H5O2 -> 2*CH3CHO + 2*HO2            ; 6.800E-14, 0
+[lc2h5o2_ch3]  C2H5O2 + CH3O2 -> CH3CHO + CH2O + 2*HO2        ; 2.500E-14, 0
+[lc2h5o2_ho2]  C2H5O2 + HO2 -> C2H5OOH                        ; 7.500E-13, 700
+[lc2h5ooh_a]   C2H5OOH + OH -> C2H5O2                         ; 1.900E-12, 190
+[lc2h5ooh_b]   C2H5OOH + OH -> CH3CHO + OH                    ; 8.010E-12, 0
+[lch3cho_oh]   CH3CHO + OH -> 0.5*CH3CO3 + 0.5*CH3O2 + 0.5*CO ; 4.630E-12, 350
+[lch3cho_no3]  CH3CHO + NO3 -> CH3CO3 + HNO3                  ; 1.400E-12, -1900
+[lch3co3_no]   CH3CO3 + NO -> CH3O2 + NO2                     ; 8.100E-12, 270
+[lch3co3_ch3]  CH3CO3 + CH3O2 -> CH2O + CH3O2 + HO2           ; 2.000E-12, 500
+[lch3co3]      CH3CO3 + CH3CO3 -> 2*CH3O2                     ; 2.900E-12, 500
+[lch3coch3_a]  CH3COCH3 + OH ->  CH3CO3 + CH2O                ; 1.330e-13, 0
+[lch3coch3_b]  CH3COCH3 + OH ->  CH3CO3 + CH2O                ; 3.820e-11, -2000
+[lroho2_no]    ROHO2 + NO -> CH3CHO + CH2O + NO2              ; 2.700E-12, 360
+[lroho2_ho2]   ROHO2 + HO2 -> CH3CHO + CH2O + OH              ; 1.500E-13, 1300
+[lroho2_ch3o2] ROHO2 + CH3O2 -> CH3CO + 2*CH2O + HO2          ; 1.000E-13, 0
+[lisopo2_no]   ISOPO2 + NO -> MVKMACR + CH2O + NO2            ; 2.700E-12, 360
+[lisopo2_ho2]  ISOPO2 + HO2 -> C2H5OOH + 2*CO                 ; 2.050E-13, 1300
+[lisopo2_ch3]  ISOPO2 + CH3O2 -> MVKMACR + 2*CH2O + 2*HO2     ; 1.000E-13, 0
+[lmvkmacr_o3]  MVKMACR + O3 -> 0.5*CH3CO3 + CH2O + CH3O2      ; 8.500E-16, -1520
+[lmvkmacr_oh]  MVKMACR + OH -> MVKO2                          ; 2.600E-12, 610
+[lmvko2_no]    MVKO2 + NO -> 0.5*CH3CO3 + CH2O + NO2          ; 2.700E-12, 360
+[lmvko2_ho2]   MVKO2 + HO2 -> C2H5OOH + CO                    ; 1.820E-13, 1300
+
+* UCI heterogeneous reactions has problems with E3SM MAM4 aerosols
+* comment out for now. To be fixed with the aerosol team.
+*[ucih1]  N2O5  -> 2*HNO3                          ; 0.05E+00
+*[ucih2]  NO3   -> HNO3                            ; 0.03E+00
+*[ucih3]  HO2   ->                                 ; 0.10E+00
+
+[usr_e90]  E90 ->                                 ; 1.286E-7, 0.
+
+* MAM4 chemistry below
+[ldms_oh]     DMS + OH -> SO2                    ; 9.600E-12, -234
+[usr_DMS_OH]  DMS + OH ->  .5 * SO2 + .5 * HO2
+[usr_SO2_OH]  SO2 + OH -> H2SO4
+[ldms_no3]    DMS + NO3 -> SO2 + HNO3            ; 1.9e-13,  520.
+ End Reactions
+
+ Ext Forcing
+      NO
```

Comment:
I am not sure why NO is needed here, but it's safer not to change it at this point.

Reply:
This is *very* old code from CESM.  It may be activating the code for providing NOx emissions from lightning, so be careful about removing it.  It also used to be connected to emissions from aircraft, although I think that got taken out at some point.   

Because the effects of this part of the code is opaque and/or hidden, it has led to big mistakes over the years.  It really should be ripped out and replaced with namelist options, especially since it isn't directly related to the chemical mechanism.

Reply:
I think we can remove it as well as for other ones. By default, this chem mech should not have elevated NO emissions. We can always added NO  elevated emission in the namelist.

Reply:
The elevated NOx emissions from lightning and aviation sources are required for chemistry. The 3D aviation NOx emission is read in from an input file and assigned to NO2 via namelist. The opaque part to me is whether this empty NO Ext Forcing is needed for the online calculation of lightning NOx emission. That's why I recommended we leave it as is for now.

Reply:
I agree with @tangq .  Lightning emissions in CESM, and hence E3SM, are calculated as a function of the cloud properties calculated by the GCM, such as the cloud-top height, so it cannot be simply read from a file.  

The entire CESM atmospheric chemistry ensemble simulations for CMIP3 were garbage because somebody removed the NO in this file.  A vast amount of time and resources were wasted, and there were no publications.